ActualIA/app
# 104 feat onboarding provider update

This feature integrate the news providers selection to the onboarding wizard and to the settings.

**To test**: 
 - Go to supabase and delete your google account to simulate a first  connection to the app or set isOnboardingRequired to true in auth_model.dart, line 34.
 - Launch the app and connect yourself with your google account
 - Complete the onboarding wizard
 - Once wizard is completed you should be on the news page 
 - then go to the settings, and click on Sources button, check that you can see the providers you entered during onboarding.
